plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -1025,7 +1056,8 @@ static bool try_connect(struct ffmpeg_output *output)
 	config.height = (int)obs_output_get_height(output->output);
 	config.format = obs_to_ffmpeg_video_format(
 			video_output_get_format(video));
-
+	config.number_audio_mixes = (int)obs_data_get_int(settings, "number_mixes");
+	config.audio_tracks = (int)obs_data_get_int(settings, "audio_tracks");
 	if (format_is_yuv(voi->format)) {
```

Comment:
Consider adding a blank line above this line (after your additions) to replace the lost blank line between the `config` assignments and the `if` statement.